test/e2e/density.go

```diff
@@ -206,12 +207,18 @@ var _ = Describe("Density", func() {
 		}
 	})
 
-	It("should allow starting 100 pods per node", func() {
-		RCName = "my-hostname-density100-" + string(util.NewUUID())
-		RunRC(c, RCName, ns, "dockerfile/nginx", 100*minionCount)
-	})
+	for _, count := range []int{30, 50, 100} {
+		name := fmt.Sprintf("should allow starting %d pods per node", count)
+		if count > 30 {
+			name = "[Skipped] " + name
```

Comment:
Can you please add a comment that tests with "Skipped" in name will be skipped when running e2e without --ginkgo.focus flag? For me it's not obvious why you add this prefix.